lively.morphic/rendering/renderer.js

```diff
@@ -1493,7 +1513,7 @@ export default class Renderer {
    */
   renderTextAndAttributes (node, morph) {
     const textNode = node.querySelector(`#${morph.id}textLayer`);
-
+    if (!textNode) return;
```

Comment:
Just for safety - you have tried searching, expanding snippets, changing font colors, font sizes, copying a text,... after this change? So basically the whole palette of what the TextMorph has to offer?

Reply:
Basically. Discovered no issues so far.